templates/client-clusterrole.yaml

```diff
@@ -10,7 +10,6 @@ metadata:
     release: {{ .Release.Name }}
 {{- if (or .Values.global.enablePodSecurityPolicies .Values.global.bootstrapACLs) }}
 rules:
-{{- end }}
```

Comment:
This seemed to be in a weird place. It was previously like:
```
if a or b {
  rules:
}
if a {
- rule a
}
if b {
- rule b
}
```

Whereas this made more sense to me:
```
if a or b {
  rules:
  if a {
  - rule a
  }
  if b {
  - rule b
  }
}
```

Now it reads
if a or b {
  rules:
  if a {
  - rule a
  }
  if b {
  - rule b
  }
} else {
  rules: []
}